### ISSUE TITLE
[terra-functional-testing] Fix packageName in reporter output file 

### ISSUE THREAD

**Issue by StephenEsser** [cerner/terra-toolkit#597]
# Bug Report

## Description

When run on Jenkins CI the spec reporter is generating the packageName as `app`. This function will need to make the necessary changes to ensure the packageName is parsed correctly when run on CI builds:

https://github.com/cerner/terra-toolkit/blob/05c4503e8f0b872c0a18c38ad8f49cbee3094013/packages/terra-functional-testing/src/reporters/spec-reporter/wdio-spec-reporter.js#L118-L125

## Steps to Reproduce

1. Choose a project running terra-functional-testing
2. Run the project on CI (in this case our internal Jenkins CI)
3. View the reporter artifacts from the Jenkins build
4. Observe the spec reporter outputs a file that contains a packageName key. This key should be the package name.

## Additional Context / Screenshots

On CI the project is likely being run in a directory that is colliding with the split operator. Either because it includes the `packages` string or because the current working directory is not what we are expecting. 

## Expected Behavior

The packageName should be the correct package name.

## Possible Solution

--
